types/jasmine/index.d.ts

```diff
@@ -765,44 +765,44 @@ declare namespace jasmine {
          * Expect a promise to be pending, i.e. the promise is neither resolved nor rejected.
          * @param expectationFailOutput
          */
-        toBePending(expectationFailOutput?: any): Promise<void>;
+        toBePending(expectationFailOutput?: any): PromiseLike<void>;
```

Comment:
Seems like changing the return types of methods to `PromiseLike` actually makes them less easy to use, not more lax.

Reply:
So what happened is, when the TS team made the change to remove the ts3.1/ directory and move files outward, some repositories (aka Jasmine typings) had some changes in the outer level index.d.ts that weren't reflected in ts3.1/

This is what happened here, you can see the original commit that created these typings in the original post.

I looked at the source code for Jasmine. It uses Promise.then(success, failure) callback style. I don't see Jasmine using the Promise.then(success).catch(failure) callback style at all.

I think PromiseLike is fine here.